server/src/lib/model/leaderboard.ts

```diff
@@ -4,6 +4,14 @@ import { getLocaleId } from './db';
 import { getConfig } from '../../config-helper';
 import lazyCache from '../lazy-cache';
 import { getMySQLInstance } from './db/mysql';
+import Bucket from '../bucket';
+import { S3 } from 'aws-sdk';
+import { AWS } from '../aws';
+import Model from '../model';
+
+const s3: S3 = AWS.getS3();
+const model: Model = new Model();
```

Comment:
explicit type annotation isn't necessary here or in the line above I think, as they can be inferred

Reply:
sorry?

Reply:
Sorry, I'll try again:

TypeScript can infer the type of `s3` from the `AWS.getS3()` call, so you don't have to explicitly write `s3: S3` (and same for `model`).

Reply:
:thumbsup: